core/blockly.js

```diff
@@ -27,7 +27,7 @@
 /**
  * The top level namespace used to access the Blockly library.
  * @namespace Blockly
- */
+ **/
```

Comment:
The extra asterisk appears in other places where we have @namespace annotations, but not in other multiline comments, so ???